src/agent/rustjail/src/cgroups/fs/mod.rs

```diff
@@ -923,12 +923,12 @@ pub fn get_mounts() -> Result<HashMap<String, String>> {
     let paths = get_paths()?;
 
     for l in fs::read_to_string(MOUNTS)?.lines() {
-        let p: Vec<&str> = l.split(" - ").collect();
+        let p: Vec<&str> = l.splitn(2, " - ").collect();
         let pre: Vec<&str> = p[0].split(' ').collect();
         let post: Vec<&str> = p[1].split(' ').collect();
 
         if post.len() != 3 {
-            warn!(sl!(), "mountinfo corrupted!");
+            warn!(sl!(), "can't parse {} line {:?}", MOUNTS, l);
```

Comment:
I find the new message a bit too generic, and I think we should not use contracted forms like "can't".

What about: "cannot extract mountpoint from {}, the line read was {:?}"?

Reply:
Really, Christophe?  You're going to nitpick my error message when it's replacing `mountinfo corrupted!` with no further information?

I actually thing your suggested replacement is needlessly wordy, and not really accurate, since it's not specifically about extracting the mountpoint.  Avoiding contractions is generally recommended for formal writing, and I'd argue for error messages, succinctness is more important than formality.